classes/torizon-signed.bbclass

```diff
@@ -13,7 +13,25 @@ inherit tdx-signed
 #   the system will be capable of booting from a legacy ostree deployment (based
 #   on hardlinks).
 #
-# - cfs-signed: TBD (not yet implemented)
+# - cfs-signed: Enable the actual ostree commit signing at build time (plus the
+#   addition of the composefs digest into the commit metadata) and the signature
+#   verification at runtime.
 #
-# TODO: Get rid of remaining uses of override "torizon-signed".
+# TODO: DISTROOVERRIDES:append = ":cfs-support:cfs-signed"
 DISTROOVERRIDES:append = ":cfs-support"
+
+# Variables a user will likely want to set up:
+#
+# - CFS_GENERATE_KEYS: Set to "1" or "0" to enable or disable the composefs key
+#   pair generation at build time.
+#
+# - CFS_SIGN_KEYDIR: Directory where the composefs keys are kept (retrieved from
+#   or generated into when not found (if the key generation is enabled)).
+#
+# - CFS_SIGN_KEYNAME: Base name of the key files; the file holding the secret
+#   key will be named ${CFS_SIGN_KEYNAME}.sec and the one with the public key
+#   will be ${CFS_SIGN_KEYNAME}.pub.
+#
+CFS_GENERATE_KEYS = "1"
+CFS_SIGN_KEYDIR = "${TOPDIR}/keys/ostree"
+CFS_SIGN_KEYNAME = "cfs-dev"
```

Comment:
Are these variables intended to be optionally override by the user? The above comment makes it sound like a user could override these, but a hard assignment is used here which suggests that maybe these are not meant to be changed by the user.

Reply:
Good catch, I missed the question marks!

Reply:
Fixed